### DISCUSSION TITLE
My bot connecting to voice channel but nothing happens when i use my local lavalink server

### DISCUSSION THREAD

**Discussion by ahmettkoca** [DSharpPlus/DSharpPlus#2005] [Support] [answered]
Hi, i have two issues. one is i am currently trying to run lavalink server on my local and my bot couldn't establish connection to my local lavalink server because it seems dsharpplus.lavalink library is only supports lavalinkV3 (i have the latest version). So i fixed that by running lavalink server v3 instead of v4 and it works. Connection established but when i executed a play command lavalink finds the song, my bot connects to voice channel but nothing happens. It doens't play. Might i did something wrong? If i connected to someone's lavalinkv3 server it working. Any ideas of why it doesn't play on my local?

**Comment by akiraveliara** [comment, marked as answer]
the lavalink extension is defunct, we recommend Lavalink4NET instead.
it's marked as obsolete in nightly builds and will not be part of DSharpPlus v5.
